qa/tasks/cephfs/test_volumes.py

```diff
@@ -465,9 +463,8 @@ def test_nonexistent_subvolume_snapshot_rm(self):
         except CommandFailedError as ce:
             if ce.exitstatus != errno.ENOENT:
                 raise
-
-        # force remove snapshot
-        self._fs_cmd("subvolume", "snapshot", "rm", self.volname, subvolume, snapshot, "--force")
```

Comment:
As you're removing this, you'll need to write a separate test that `fs subvolume snapshot rm --force` works as expected.